src/Illuminate/Queue/CallQueuedHandler.php

```diff
@@ -230,10 +230,10 @@ protected function handleModelNotFound(Job $job, $e)
         }
 
         if ($shouldDelete) {
-            return $job->delete();
+            $job->delete();
```

Comment:
Hi @johnathanmdell 

I don't know if you are going to send it to master as suggested by @GrahamCampbell , but removing the return here also changes the code execution.

Before this method would not continue execution if `$shouldDelete` evaluated to true, skipping the call for `$job->fail($e)`.